src/System.Transactions/tests/System.Transactions.Tests.csproj

```diff
@@ -15,6 +15,10 @@
     <Compile Include="TransactionScopeTest.cs" />
     <Compile Include="AsyncTest.cs" />
     <Compile Include="EnlistTest.cs" />
+    <Compile Include="CloneTxTests.cs" />
+    <Compile Include="LTMEnlistmentTests.cs" />
+    <Compile Include="HelperFunctions.cs" />
+    <Compile Include="TestEnlistments.cs" />
```

Comment:
Hmm. git status still shows lowercase. How do I fix this?


---
In reply to: [72360242](https://github.com/dotnet/corefx/pull/10350#discussion_r72360242) [](ancestors = 72360242,72359008)